js/react_native/ios/Podfile

```diff
@@ -40,11 +48,10 @@ target 'OnnxruntimeModuleTest' do
 end
 
 post_install do |installer|
-    installer.generated_projects.each do |project|
-        project.targets.each do |target|
-            target.build_configurations.each do |config|
-                config.build_settings['IPHONEOS_DEPLOYMENT_TARGET'] = '15.1'
-            end
-        end
-    end
-end
+  # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
+  react_native_post_install(
+    installer,
+    :mac_catalyst_enabled => false,
+  )
+  __apply_Xcode_12_5_M1_post_install_workaround(installer)
```

Comment:
The workaround was for Xcode 12.5 and Apple Silicon machines.  But we are using XCode 14.3. Do we still need it? What error did you see? Please make a note here.

Reply:
Same it is copied from [facebook](https://react-native-community.github.io/upgrade-helper/?from=0.69.3&to=0.70.15&package=onnxruntime-android&name=onnxruntime). Will be removed in the future [updates](https://react-native-community.github.io/upgrade-helper/?from=0.69.3&to=0.76.6&package=onnxruntime-android&name=onnxruntime).